### ISSUE TITLE
Keyboard in numbers are automatically changes to all keyboard.

### ISSUE THREAD

**Issue by abhi22211** [david-legend/otp_textfield#23]
Keyboard in numbers are automatically changes to all keyboard. Even after applying KeyboardType.number. Not working properly at all. If I have set to KeyboardType.number then it should not change.

**Comment by adebola-duf** [comment]
I don't think this is a problem with this package. Because even when I implemented mine, I was having the same issue. I think the issue comes from when you are changing the focus of each textfield after filling a character.

**Comment by sunil-singh-chaudhary** [comment]
did you find any solution same problem occuring to me

**Comment by abhi22211** [comment]
Not yet

**Comment by sunil-singh-chaudhary** [comment]
ok i remove this lib and use pinput which is much better and easy to use
